lib/index.js

```diff
@@ -1178,8 +1178,14 @@ exposeMethod('consume');
  * not provide a new stream for you to push values onto, and it will unsubscribe
  * as soon as it has a single error, value or nil from the source.
  *
- * You probably won't need to use this directly, but it is used internally by
- * some functions in the Highland library.
+ * You can use this method to integrate with other libraries that expect a
+ * function that takes a node style callback as its parameter. Example of such
+ * libraries are promise libraries.
+ *
+ * You might want to chain `pull` after a [`collect`](#collect) too send all
```

Comment:
"too" -> "to".

Also, using `pull` doesn't technically end the (collected) stream, so I would drop the clause at the end. It's just not a problem because `collect` will end all upstream streams.